src/main/resources/SnakemakeBundle.properties

```diff
@@ -182,6 +185,22 @@ PARSE.expected.indent=Indent expected
 PARSE.incorrect.unindent=Unindent does not match any outer indentation level.
 PARSE.eof.docstring=Docstring at end of file does not precede any statement
 PARSE.rule.expected.rule.commend.to.docstring=Expecting rule keyword, comment or docstrings inside a rule definition.
+PARSE.use.with.not.allowed=Keyword 'with' in 'use rule' statement is not allowed in combination with rule pattern '*'.
+PARSE.use.double.mult.sign=Expecting rulename modifying pattern (e.g. modulename_*) after 'as' keyword
+PARSE.use.requires.wildcard=Multiple rules in 'use rule' statement but name modification ('as' statement) does not contain a wildcard '*'.
+PARSE.use.rule.keyword.expected=Expecting keyword 'rule' after keyword 'use'
+PARSE.use.names.expected=Expecting '*', rule name, or rule listing (comma separated) after 'use rule' statement
+PARSE.use.few.names.from.current.module='use rule' statement from rule in the same module must declare a single rule but multiple rules are declared.
+PARSE.use.unexpected.names.separator=Unexpected token in comma separated list of rules within 'use rule' statement.
+PARSE.use.wildcard.in.names.list=Wildcard '*' can't be used with rules names List
+PARSE.use.unexpected.list.ending=Expecting list of rules in 'use rule' statement to end with keyword 'from'.
+PARSE.use.expecting.module.name=Expecting module name after 'from' keyword in 'use rule' statement.
+PARSE.use.with.missed=':' can be placed only after 'with' keyword, which is missed
```

Comment:
Some parsing errors ends with '.' here, some not, please using single style. If I'm not mistaken normally we do not use '.' for parsing error messages in python and other languages